### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by Eclextic** [hyprwm/aquamarine#1]
So if I understood this project correctly, it seems like we should be able to create Wayland applications using Aquamarine as a wrapper.

Does that also mean that we are going to get some documentation?
Could prove helpful if this project is supposed to become serious or mature over the years...

**Comment by vaxerski** [comment]
Yes. Aquamarine provides you with a foundation to make an app run independently on wayland or (soon) a DRM session.

Documentation will be there once aq is ready for use.

**Comment by Eclextic** [comment]
Alright, I will keep this then open until documentation is added so that others can see this.

**Comment by Eclextic** [comment]
Also further (slightly irrelevant) question: Is the doc going to be on Hyprland's site or on GitHub's Wiki feature or somewhere else?

**Comment by vaxerski** [comment]
I might make a subdomain under hyprland.org

**Comment by Edgars-Cirulis** [comment]
> Yes. Aquamarine provides you with a foundation to make an app run independently on wayland or (soon) a DRM session.
> 
> Documentation will be there once aq is ready for use.

Its not yet ready for use why then Hyprland uses it? "Wondering"

**Comment by vaxerski** [comment]
you're being purely annoying for the sake of being an annoying child. If you continue doing that, I will ban you from this org.

The comment is from way before aq for hyprland was merged.

**Comment by Edgars-Cirulis** [comment]
> you're being purely annoying for the sake of being an annoying child. If you continue doing that, I will ban you from this org.
> 
> The comment is from way before aq for hyprland was merged.

Ehh, it was just a question because I'd see aquamarine documentation so I can understand it instead of reading whole source code.
